Stylesheets/Authentication/signup.css

```diff
@@ -0,0 +1,86 @@
+/* Sign Up Styles */
+
+.glass__form {
+    margin: 2rem;
+    padding: 2rem;
+}
+
+.glass__input {
+    background-color: var(--component-grey-01);
+    color: black;
+    font-weight: 500;
+}
+
+.glass__input::placeholder {
+    color: #414141;
```

Comment:
you can use variable here for better readability

Reply:
Thank You. Will be adding that color to my colors stylesheet and updating it.